packages/timere-parse/timere-parse.0.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Timere natural language parsing component"
+description: """
```

Comment:
If you have time, even a brief description would be appreciated. It's important to provide hints to other users of opam as they browse

Reply:
Ah yep. I've added a short one for now.

Reply:
thanks!